### PR TITLE
Make tables page more accessibile to screen readers

### LINKED ISSUE #37: Sites table is completely unusable with a screen reader
At the moment it is not possible for me to tell which 2FA options are available on each listed site, and navigation is difficult. This can be broken down into two related issues:

1. The list of sites is not actually a table.
2. The symbols used for indicating a 2FA option are not read by screen readers.

Screen readers can navigate a proper HTML table in any of the four directions, and with valid table row/column headers, everything should be read properly. Unfortunately right now, the lack of a real table means that I don't get any spatial information and can't tell which options are available for every site. I can probably download the files and read them that way, but it would be great if the public-facing site were also usable.

As for subissue 2, since row/column headers are readable, it would be helpful if the 2FA cells had a "yes" or "no" aria-label. The screen reader will fill in the missing information, prefixing the "yes" or the "no" with the appropriate header depending on the direction of movement. There's no need to include the name of the service or the 2FA type in the aria-label.

### PR DESCRIPTION
**Pull request by Carlgo11**
Partially resolves issue #37

* Fixes structure of document for screen readers.
* Links category buttons with tables
* Hides unnecessary data for screen readers.
* Adds "visibly hidden" `.tfa-summary` in Desktop view.

### CODE REVIEW

**Comment by Carlgo11** [comment]
I messed up the CSS for .tfa-summary. Please don't merge this yet 😬

**Comment by Carlgo11** [comment]
Could someone pretty please look at this pr? 🙏
@2factorauth/maintainers

**Review by dsoegijono — APPROVED** [review]

**Pull request merged**
